### ISSUE TITLE
bug(?): Logs number in VictoriaLogs is less than in Loki

### ISSUE THREAD

**Issue by arseny-zinchenko** [VictoriaMetrics/VictoriaMetrics#7266]
**Chart name and version**

- VictoriaLogs: `v0.35.0`
- Grafana VictoriaLogs Data source version: `0.5.0`

**Describe the bug**
In Grafana, when selecting the same logs using similar queries, the resulted logs number differs.

_I am not sure if it's a bug, or I'm doing/understanding something wrong._

**Loki**
Query: `{app="backend-api", namespace="prod-backend-api-ns"} `
Result is `logs Total: 2.89 K`:

![image](https://github.com/user-attachments/assets/33c098fa-72b2-4e24-9d56-e7520be2de15)

**VictoriaLogs**
Query: `app:="backend-api" namespace:="prod-backend-api-ns"`
Result is `info Total: 42 warning Total: 1`:

![image](https://github.com/user-attachments/assets/f72f80c0-a813-4b7f-bcde-94fea3adeed4)

Also, wondering why in the VMLogs results, they are displayed only at the begging of the **Logs volume**, while in Loki - the whole "**Logs volume**" is filled.
I've tried playing with the `_time`, but no luck.

**Comment by hagen1778** [comment]
@arseny-zinchenko could you please elaborate on how you push these logs to both solutions? Any way how we can reproduce the same issue locally, maybe you have docker-compose file for this type of installation?

**Comment by AndrewChubatiuk** [comment]
related issue https://github.com/VictoriaMetrics/victorialogs-datasource/issues/91

**Comment by arseny-zinchenko** [comment]
@hagen1778 

Logs are pushed to the VMLogs with Promtail:

```
promtail:
  config:
    clients:
      - url: http://atlas-victoriametrics-loki-gateway/loki/api/v1/push
      - url: http://atlas-victoriametrics-victoria-logs-single-server:9428/insert/loki/api/v1/push?_stream_fields=namespace
```

Promtail is installed from our Helm chart as dependency:

```
- name: promtail
  version: ~6.16.0
  repository: https://grafana.github.io/helm-charts
```

In Grafana VMLogs data source is installed in this way:

```
  grafana:
    enabled: true
   ...
    env:
      GF_PLUGINS_ALLOW_LOADING_UNSIGNED_PLUGINS: "victorialogs-datasource"
    ...
    plugins:
     ...
      - https://github.com/VictoriaMetrics/victorialogs-datasource/releases/download/v0.5.0/victorialogs-datasource-v0.5.0.zip;victorialogs-datasource
   ...
    additionalDataSources:
      - name: VictoriaLogs
        type: victorialogs-datasource
        access: proxy
        url: http://atlas-victoriametrics-victoria-logs-single-server:9428
```

**Comment by hagen1778** [comment]
This issue is related to bug in datasource plugin parsing logic and was addressed in https://github.com/VictoriaMetrics/victorialogs-datasource/pull/95.
The fix will be available in the next datasource release.

**Comment by hagen1778** [comment]
Re-opening the issue because datasource release didn't happen yet.
@dmitryk-dk do you have plans to make a new release soon?

**Comment by dmitryk-dk** [comment]
> Re-opening the issue because datasource release didn't happen yet. @dmitryk-dk do you have plans to make a new release soon?

I can make it today. Thought that we could have any other bug fixes or features. So will do it today

**Comment by dmitryk-dk** [comment]
Close the issue. The issue should be resolved in the released [v0.6.2](https://github.com/VictoriaMetrics/victorialogs-datasource/releases/tag/v0.6.2) version
